### PR TITLE
Updates and fixes!

### PR DESCRIPTION
**Pull request by WaelAbedElal**
Trying to contribute to and improve the servers! 
Other ideas to come! :) 
- HaRCoR7

### CODE REVIEW

**Comment by AleksCore** [comment]
Yay

**Comment by EsDeKa** [comment]
Nice work!

The mtaserver.conf file here is just an example to get your own local server running, it's different from what the actual files are 

I enabled autostart for race assist in /config instead, easier

**Comment by JarnoVgr** [comment]
Thanks for the PR! :)

File: resources/[gameplay]/gcshop/horns/horns_c.lua [hunk 1]
```diff
@@ -711,7 +711,7 @@ function createSoundForCar(car, horn)
 			guiSetAlpha(icon[car], globalalpha)
 			guiSetSize(icon[car], relx, rely, false)
 			if(screenX and screenY) then
-				guiSetPosition(icon[car], screenX, screenY, false)
+				guiSetPosition(icon[car], screenX-50, screenY+50, false)
```
Comment (AleksCore) [inline resources/[gameplay]/gcshop/horns/horns_c.lua]: probably need to move a bit more, sadly I can't experiment on this at the moment as I have no local server etc

**Pull request merged**